myko_api_client/utils.py

```diff
@@ -0,0 +1,75 @@
+import json
+import logging
+import os
+import uuid
+
+import requests
+from d3a_interface.constants_limits import JWT_TOKEN_EXPIRY_IN_SECS
+from d3a_interface.utils import RepeatingTimer
+
+
+def retrieve_jwt_key_from_server(domain_name):
```

Comment:
You can install `d3a-api-client` as a library and simply import if from there as it seems to be the exact same implementation.  Same goes for the rest of utility methods and classes as part of this module.